README.md

```diff
@@ -53,5 +53,19 @@ See [contrib/tools/workflowcheck](contrib/tools/workflowcheck) for a tool to det
 ## Contributing
 We'd love your help in making the Temporal Go SDK great. Please review our [contribution guidelines](CONTRIBUTING.md).
 
+## Go build and run tags
+
+Prior to SDK version v1.26.0 our protobuf code generator allowed invalid UTF-8 data to be stored as proto strings. This isn't actually allowed by the proto3 spec, so if you're building our SDK manually and think you may store arbitrary binary data in our strings you should set `-tags protolegacy` when building against our SDK.
```

Comment:
"building our SDK manually" can we say  "using our SDK" with Go everything is always distributed as source right so the users are always building our SDK when they include it as a depdency?

Reply:
You're right, that's poor phrasing on my part. That's fixed in the latest commit